src/main/kotlin/de/bund/digitalservice/useid/metrics/MetricsService.kt

```diff
@@ -0,0 +1,30 @@
+package de.bund.digitalservice.useid.metrics
+
+import de.bund.digitalservice.useid.tenant.PARAM_NAME_TENANT_ID
+import io.micrometer.core.instrument.Counter
+import io.micrometer.core.instrument.Metrics
+import org.springframework.stereotype.Service
+
+@Service
+class MetricsService {
```

Comment:
An improvement is to keep a record of counters per name and tag list. This way, we would create fewer counters objects per request.